js/tests/models/contact_model.js

```diff
@@ -27,4 +27,11 @@ describe('contactModel', function() {
 		var d = contact.getISODate(new Date('2016-09-01T09:07:05Z'));
 		expect(d).to.equal('20160901T090705Z');
 	});
+
+	it('should fix invalid group array', function() {
+		var contact = new $Contact({displayName: 'test'});
+		contact.categories(['Test 1', 'Test 2\,Test 3']);
+		var categories = contact.categories();
+		expect(categories).to.equal(['Test 1', 'Test 2', 'Test 3']);
```

Comment:
@ChristophWurst This doesn't work. Any idea why?
`expected [ 'Test 1', 'Test 2', 'Test 3' ] to equal [ 'Test 1', 'Test 2', 'Test 3' ]`